qa/vagrant/src/test/resources/packaging/utils/utils.bash

```diff
@@ -415,6 +415,50 @@ stop_elasticsearch_service() {
     fi
 }
 
+# the default netcat packages in the distributions we test are not all compatible
+test_port() {
+    local host="$1"
+    local port="$2"
+    cat < /dev/null > "/dev/tcp/$host/$port"
+}
+
+describe_port() {
+    local host="$1"
+    local port="$2"
+    run test_port "$host" "$port"
+    if [ "$status" -eq 0 ]; then
+        echo "port $port on host $host is open"
+    else
+        echo "port $port on host $host is not open"
+    fi
+}
+
+debug_collect_logs() {
+    local es_logfile="$ESLOG/elasticsearch.log"
+    local system_logfile='/var/log/messages'
+
+    if [ -e "$es_logfile" ]; then
+        echo "Here's the elasticsearch log:"
+        cat "$es_logfile"
+    else
+        echo "The elasticsearch log doesn't exist at $es_logfile"
+
+	if [ -e "$system_logfile" ]; then
```

Comment:
Maybe we can always print both logs? We have at least 1 test with 2 nodes and in this case it would only prints the first one.

Reply:
I like that idea, I'll make it print both